src/core_write.cpp

```diff
@@ -106,7 +106,7 @@ std::string ScriptToAsmStr(const CScript& script, const bool fAttemptSighashDeco
                         const unsigned char chSigHashType = vch.back();
                         if (mapSigHashTypes.count(chSigHashType)) {
                             strSigHashDecode = "[" + mapSigHashTypes.find(chSigHashType)->second + "]";
-                            vch.pop_back(); // remove the sighash type byte. it will be replaced by the decode.
+                            vch.pop_back(); // Remove the sighash type byte. It will be replaced by the decode.
```

Comment:
Capitalization changes are unlikely to get consensus ACK, so please skip the changes made to this file.

Reply:
While this change may appear to be purely cosmetic, I do think that it's okay since it is being included with a number of other changes and because the overall intention is to ensure greater consistency.

Reply:
I didn't change this one. Let me know if you still need this changed.

Reply:
I would revert this change in order to maximize the chance of merge.

Reply:
Reverted.